test/wait/contrail.go

```diff
@@ -5,9 +5,10 @@ import (
 	"strings"
 	"time"
 
+	"github.com/operator-framework/operator-sdk/pkg/test"
+
 	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
 	"github.com/Juniper/contrail-operator/test/logger"
-	"github.com/operator-framework/operator-sdk/pkg/test"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
These ones should be one section higher.

Reply:
fixed